packages/elements/src/photon-authorized/index.tsx

```diff
@@ -46,30 +51,27 @@ export const PhotonAuthorized = (p: { children: JSXElement; permissions?: Permis
     );
   });
 
+  const canAccess = createMemo(() => authenticated() && inOrg() && hasPermission());
```

Comment:
We want the `hasPermission` to be a different error message, something like "You are not authorized to prescribe"